esp/platform/esp.hpp

```diff
@@ -197,9 +199,15 @@ interface IEspContext : extends IInterface
 
     virtual CTxSummary* queryTxSummary()=0;
     virtual void addTraceSummaryValue(unsigned logLevel, const char *name, const char *value, const unsigned int group = TXSUMMARY_GRP_CORE)=0;
+    virtual void addTraceSummaryValue(unsigned logLevel, const char *name, const char *value, const char* otName, unsigned group = TXSUMMARY_GRP_CORE)=0;
```

Comment:
I can infer otName signifies "alternate name to be used by opentelemetry", but I'm not sure it's as obvious and self documenting as it could be. Can we use a self documenting name which ensures future developer will know exactly what this param is for?
Also, is this param only for OpenTelemetry?